Development/src/pages/receivers/ReceiversShow.js

```diff
@@ -0,0 +1,538 @@
+                        label="Sender"
+                        source="$staged.sender_id"
+                        reference="senders"
+                        link="show"
+                    >
+                        <ChipConditionalLabel source="label" />
+                    </ReferenceField>
+                )}
+                <BooleanField
+                    label="Master Enable"
+                    source="$staged.master_enable"
+                />
+                <TextField label="Mode" source="$staged.activation.mode" />
+                <TAIField
+                    label="Requested Time"
+                    source="$staged.activation.requested_time"
+                    mode="$staged.activation.mode"
+                />
+                <TAIField
+                    label="Activation Time"
+                    source="$staged.activation.activation_time"
+                />
+                <TextField label="Transport Type" source="$transporttype" />
+                <ArrayField
+                    label="Transport Parameters"
+                    source="$staged.transport_params"
+                >
+                    <ReceiverTransportParamsCardsGrid
+                        record={controllerProps.record}
+                    />
+                </ArrayField>
+                <TransportFileViewer endpoint="$staged.transport_file.data" />
+                <JSONViewer endpoint="$staged" />
+            </SimpleShowLayout>
+        </ShowView>
+    );
+};
+
+const ConnectionManagementTab = ({
+    controllerProps,
+    receiverData,
+    ...props
+}) => {
+    const refresh = useRefresh();
+    const [filter, setFilter] = useState({
+        transport: get(receiverData, 'transport'),
+    });
+    const [paginationCursor, setPaginationCursor] = useState(null);
+    // As the paginationCursor variable has not changed we need to force an update
+    const [seed, setSeed] = useState(Math.random());
+
+    const { data, error, loaded } = useGetList({
+        ...props,
+        filter,
+        paginationCursor,
+        resource: 'senders',
+        seed,
+    });
+
+    const nextPage = label => {
+        setPaginationCursor(label);
+        setSeed(Math.random());
+    };
+
+    const changeFilter = (filterValue, name) => {
+        let currentFilter = filter;
+        if (filterValue) {
+            currentFilter[name] = filterValue;
+        } else {
+            delete currentFilter[name];
+        }
+        setFilter(currentFilter);
+        setPaginationCursor(null);
+        setSeed(Math.random());
+    };
+
+    // receiverData initialises undefined, update when no longer null
+    useEffect(() => {
+        if (receiverData !== null)
+            setFilter({ transport: get(receiverData, 'transport') });
+    }, [receiverData]);
+
+    const connect = (senderID, receiverID, endpoint) => {
+        MakeConnection(senderID, receiverID, endpoint, props).then(() => {
+            refresh();
```

Comment:
New call to `refresh()` here?

Reply:
Refresh is now a react hook provided by react-admin. As MakeConnections is not a react functional component, we cannot use hooks within it.